packages/client/modules/teamDashboard/components/AgendaItem/AgendaItem.tsx

```diff
@@ -238,7 +238,9 @@ const AgendaItem = (props: Props) => {
         />
       </AgendaItemStyles>
       {tooltipPortal(
-        pinned ? `Unpin "${content}" from every check-in` : `Pin "${content}" to every check-in`
+        pinned
+          ? `Unpin this agenda topic from every check-in`
```

Comment:
+1 why remove the "content" bit out of interest? The tooltip updates the position so we can always see it. I prefer being able to see the content of the tooltip I'm pinning so I'm sure I'm pinning the right one

Reply:
I find it redundant & too verbose? I'm trying to follow a "industry standard" (if that's a thing😂). For example, in Arc tab management:

<img width="269" alt="Screenshot 2024-09-18 at 10 39 04" src="https://github.com/user-attachments/assets/eadc0309-8ea2-447a-bd95-91ef518a8ae4">

Notice here it says "archive this tab", not "archive 'GitHub'"